File: frontend/app/api/run/reference/2.py
def main():
    # Read initial queue elements
    queue = list(map(int, input().split()))

    # Read element to enqueue
    x = int(input())
    queue.append(x)

    # Print queue after enqueue
    print(*queue)

    # Dequeue two elements (remove from front)
    for _ in range(2):
        if queue:
            queue.pop(0)

    # Print queue after dequeue
    print(*queue)


if __name__ == "__main__":
    main()

File: frontend/app/api/run/reference/2.c
#include <stdio.h>

int main() {
    int queue[1000], n = 0, x;
    
    // Read initial queue elements
    while (scanf("%d", &queue[n]) == 1) {
        n++;
        if (getchar() == '\n') break;
    }

    // Read element to enqueue
    scanf("%d", &x);
    queue[n++] = x;

    // Print queue after enqueue
    for (int i = 0; i < n; i++)
        printf("%d ", queue[i]);
    printf("\n");

    // Dequeue two elements (remove from front)
    int start = 2;
    if (start > n) start = n;

    for (int i = start; i < n; i++)
        printf("%d ", queue[i]);
    printf("\n");

    return 0;
}
